react/tic-tac-toe/src/services/MatchesService.js

```diff
@@ -1,5 +1,9 @@
+import { wrapService } from 'redux-recompose';
+
 import api from '~config/api'; // eslint-disable-line import/no-unresolved
 
-export default {
+const service = {
   getMatches: () => api.get('/matches')
 };
+
+export default wrapService(service, 'match', { getMatches: 'matches' });
```

Comment:
create this `match` in a constants file in de redux folder, something like `REDUCER_TARGET`
and the other one `matches` as a `MATCHES_TARGET` and use it anywhere you use this reducer.